File-Formats/PDF/Convert-HTML-To-PDF/WPF.md

```diff
@@ -8,69 +8,70 @@ keywords: Assemblies
 ---
 
 # Convert HTML to PDF file in WPF
-## Steps to convert Html to PDF document in WPF:
 
-Create a new WPF application project.
+The Syncfusion HTML to PDF converter is a .NET library used to convert HTML or web pages to PDF in WPF application.
+
+## Steps to convert Html to PDF document in WPF
+
+1. Create a new WPF application project.
 ![Convert HTMLToPDF WPF Step1](htmlconversion_images/WPF_Step1.png)
-Now, the project configuration window appears. Click Create button to create a new project with the default project configuration.
+In project configuration window, name your project and select Create.
 ![Convert HTMLToPDF WPF Step2](htmlconversion_images/WPF_Step2.png)
 
-Install the [Syncfusion.HtmlToPdfConverter.WPF](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.WPF) NuGet package as a reference to your WPF application [NuGet.org](https://www.nuget.org/)
+2. Install the [Syncfusion.HtmlToPdfConverter.WPF](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.WPF) NuGet package as a reference to your WPF application [NuGet.org](https://www.nuget.org/).
 ![Convert HTMLToPDF Docker Step3](htmlconversion_images/WPF_Step3.png)
 
-Include the following namespaces in the MainWindow.xaml.cs file.
-
-{% highlight c# tabtitle="C#" %}
-
-using System;
-using System.IO;
-using System.Windows;
-using Syncfusion.Pdf;
-using Syncfusion.HtmlConverter;
-
-{% endhighlight %}
-
-Add a new button in MainWindow.xaml to convert Html to Pdf document as follows.
-
-{% highlight c# tabtitle="C#" %}
-
-<Grid HorizontalAlignment="Left" Margin="0,0,0,-0.333" Width="793">
-<Button Content="Convert Html ot PDF" HorizontalAlignment="Left" Margin="318,210,0,0" VerticalAlignment="Top" Width="166" Click=" btnCreate_Click " Height="19"/>
-<TextBlock HorizontalAlignment="Left" Margin="222,177,0,0" TextWrapping="Wrap" VerticalAlignment="Top" Height="17"/>
-<TextBlock HorizontalAlignment="Left" Margin="291,175,0,0" TextWrapping="Wrap" Text="Click the button to convert Htnl to PDF." VerticalAlignment="Top"/>
-
-</Grid>
-
-{% endhighlight %}
+3. Include the following namespaces in the MainWindow.xaml.cs file.
 
-Add the following code in btnCreate_Click to convert Html to PDF document.
+   {% highlight c# tabtitle="C#" %}
 
-{% highlight c# tabtitle="C#" %}
+   using System;
+   using System.IO;
+   using System.Windows;
+   using Syncfusion.Pdf;
+   using Syncfusion.HtmlConverter;
 
-// Initialize HTML to PDF converter.
-HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+   {% endhighlight %}
 
-// Convert URL to PDF.
-PdfDocument document = htmlConverter.Convert("https://www.syncfusion.com");
+4. Add a new button in MainWindow.xaml to convert Html to Pdf document as follows.
```

Comment:
Pdf should be "PDF" and Html should be "HTML"

Reply:
Changed content